Add Summit-XL Steel

mael25 opened the pull request:
**Description**
Adds the Robotnik Summit-XL Steel robot
**This PR replaces #2969 which targeted the `master` branch**

**Tasks**
  - [x] Create Icons
  - [x] Documentation Summit-XL Steel
  - [x] Better appearance & documentation SickS300 (lidar-sensors.md)
  - [x] Update robots Makefile
  - [x] Update Changelog
  - [x] Improve appearance base & Redo pictures/icons
  - [x] Add web component
  - [x] Fix `contactProperties` issue
  - [x] Add missing sensors

<img src="https://user-images.githubusercontent.com/38250944/115353386-ba80f500-a1b8-11eb-91c6-67f812ff1925.png" width="50%">.

BenjaminDeleze reviewed (changes requested):
The plastic crate that is on the robotino plateform is a bit _in_ the conveyor belt and so it cannot move.

mael25 commented on docs/guide/summit-xl-steel.md:
```diff
@@ -0,0 +1,74 @@
+
+%robot summit-xl-steel images/robots/summit_xl_steel/model.thumbnail.png
+
+Designed by Robotnik, [Summit-XL Steel](summit-xl-steel.md) is a four-wheeled (mecanum) robot.
```
You're right, I have replaced it by the link to the webpage on the constructor website, as it is done for several other robots currently.

mael25 commented:
@BenjaminDeleze I have slightly offseted the `PlasticCrate` so that it moves on the conveyor belt. I have also added the missing sensors (`GPS` and RGBD camera, implemented as a `Camera` and a `RangeFinder`)

BenjaminDeleze reviewed (approved):
Thank you!
